deploy.sh

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env bash
+set -ex
+
+./git-version.sh
+
+# npx ng build --prod --base-href "..."
+npx ng build --prod
+npx firebase deploy
```

Comment:
We'll probably have to do some credentials juggling (or shared projects might not even be free... idk) in order to get you deploy access

Side note: Firebase hosting is super easy to set up -- def recommend trying it out (creating a new project) on your own too